world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
```

Comment:
@ergodicbreak this is the part where I am not clear on.  I would appreciate if you could clarify the following questions:

* Did you want abilities to be structured as nested dictionaries in literal format?  If so, should these be stored under one umbrella dictionary similar to the following:

abilities = {
    'kick' : {a:1,b:2},
    'hide' : {a:1,b:2},
}

* I don't understand the purpose of the __init__ files and how we use these in reference to abilities.

* Would we still need to import the abilities dictionary as required when we need to refer to values within the dictionary?  

Thanks, my apologies I know these are simple questions, I haven't been able to completely comprehend how this will work.